### PR TITLE
add a log message post ingest decision

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: functions/zoom-uploader.py [hunk 1]
```diff
@@ -94,26 +94,22 @@ def handler(event, context):
     upload_message = messages[0]
     logger.debug({
         "queue_message": {
-            "attributes": upload_message.attributes,
```
Comment (nataliemona) [inline functions/zoom-uploader.py:97]: this is always null

File: functions/zoom-uploader.py [hunk 2]
```diff
@@ -94,26 +94,22 @@ def handler(event, context):
...
             "body": upload_message.body
         }
     })
 
     try:
         upload_data = json.loads(upload_message.body)
-        logger.info({
```
Comment (nataliemona) [inline functions/zoom-uploader.py:104]: move to after ingest decision

File: functions/zoom-uploader.py [hunk 3]
```diff
@@ -142,8 +138,8 @@ def get_current_upload_count():
 
 def process_upload(upload_data):
     upload = Upload(upload_data)
-    upload.upload()
-    return upload.workflow_id
```
Comment (nataliemona) [inline functions/zoom-uploader.py:146]: this was calling the function workflow_id multiple times

File: functions/zoom-uploader.py [hunk 4]
```diff
@@ -239,10 +235,10 @@ def s3_filenames(self):
 
     @property
     def workflow_id(self):
-        if not hasattr(self, "workflow_xml"):
-            logger.warning("No workflow xml yet!")
-            return None
         if not hasattr(self, "_workflow_id"):
+            if not hasattr(self, "workflow_xml"):
```
Comment (nataliemona) [inline functions/zoom-uploader.py:239]: workflow_xml should only be checked once

File: tasks.py [hunk 5]
```diff
@@ -84,6 +84,9 @@ def production_failsafe(ctx):
     It is meant to be prepended to the execution of other tasks to force a confirmation
     when a task is being executed that could have an impact on a production stack
     """
+    if not STACK_NAME:
+        raise Exit("No STACK_NAME specified")
+
```
Comment (nataliemona) [inline tasks.py:89]: if prod identifier is an empty string and STACK_NAME is not defined, then the production failsafe will be triggered and then the user will get an error message that doesn't make it clear that the problem is that STACK_NAME is not defined

File: tasks.py [hunk 6]
```diff
@@ -338,15 +341,15 @@ def exec_on_demand(ctx, uuid, oc_series_id=None, allow_multiple_ingests=False):
 @task(help={'uuid': 'meeting instance uuid',
             'ignore_schedule': ('ignore schedule, use default series if '
                                 'available'),
-            'on_demand_series_id': ('opencast series id to use regardless of '
+            'oc_series_id': ('opencast series id to use regardless of '
```
Comment (nataliemona) [inline tasks.py:344]: change to make exec.on-demand and exec.webhook/exec.pipeline consistent

File: tasks.py [hunk 7]
```diff
@@ -1214,6 +1217,7 @@ def __publish_version(ctx, func, description):
 
 
 def __build_function(ctx, func, upload_to_s3=False):
+    print(f"Building {func} function")
```
Comment (nataliemona) [inline tasks.py:1220]: Add print statements that explain the steps and hide=1 to stop the terminal spam

File: functions/zoom-webhook.py [hunk 8]
```diff
@@ -207,6 +207,10 @@ def construct_sqs_message(payload, context, zoom_event):
     if "on_demand_series_id" in payload:
         sqs_message["on_demand_series_id"] = payload["on_demand_series_id"]
 
+    # not used in downloader or uploader but useful for cloudwatch dashboard
```
Comment (nataliemona) [inline functions/zoom-webhook.py:210]: I want to have the zoom processing times, and the size of the file available in one log message - it seems like zoom processing times are mostly impacted by file size so this is important.

**Review by lbjay — APPROVED** [review]

**Pull request merged**